### PR TITLE
Add support of EXPIRETIME and PEXPIRETIME

### PR DESCRIPTION
**Pull request by kay011**
This commit finish the  #1948 and  #1947. 
It should be noted that since the timeout precision currently supported by kvrocks is seconds, the results obtained by executing PEXPIRETIME will be different from those obtained by redis. This is my first time trying to implement a kvrocks command, please give me your code review comments. Thanks!

### CODE REVIEW

**Comment by PragmaTwice** [comment]
Thanks for your effort!

> It should be noted that since the timeout precision currently supported by kvrocks is seconds, the results obtained by executing PEXPIRETIME will be different from those obtained by redis.

If you build kvrocks with `-DENABLE_NEW_ENCODING=ON`, you can see that the precision of these expire time is in milliseconds.

File: src/storage/redis_db.cc [hunk 1]
```diff
@@ -226,6 +226,30 @@ rocksdb::Status Database::TTL(const Slice &user_key, int64_t *ttl) {
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status Database::GetExpireTime(const Slice &user_key, uint64_t *timestamp) {
```
Comment (git-hulk) [inline src/storage/redis_db.cc:228]: We can use `GetMetadata` directly instead of decoding here.

Comment (PragmaTwice) [reply src/storage/redis_db.cc:228]: IMHO it is hard to use since we need to pass RedisType.

Comment (git-hulk) [reply src/storage/redis_db.cc:228]: Yes, maybe we need to pass an empty redis type array to ignore the type check later.

Comment (PragmaTwice) [reply src/storage/redis_db.cc:228]: We can use `RedisTypes::All()` now.

Comment (git-hulk) [reply src/storage/redis_db.cc:228]: @kay011 Would you mind changing this implementation to `GetMetadata` since we can use `RedisTypes::All()` now.

Comment (kay011) [reply src/storage/redis_db.cc:228]: @git-hulk Of course, I'm thinking about how to use it more elegantly

Comment (git-hulk) [reply src/storage/redis_db.cc:228]: How about changing the parameter `type` of `Database::GetMetadata` to `RedisTypes` like ParseMetadata?

Comment (kay011) [reply src/storage/redis_db.cc:228]: @git-hulk I think that is the best solution! But the existed code will change the call parameter type. So there is a trade off.

Comment (git-hulk) [reply src/storage/redis_db.cc:228]: I think that's fine since only a few places are using this function.

Comment (kay011) [reply src/storage/redis_db.cc:228]: @git-hulk Nice, I will implement it as stated above.

**Comment by git-hulk** [comment]
@kay011 Thanks for your contribution and quick response.

**Review by git-hulk — APPROVED** [review]

**Pull request merged**